SitumWayfinding/Classes/Positioning/PositioningViewController.swift

```diff
@@ -88,6 +88,7 @@ class PositioningViewController: UIViewController, GMSMapViewDelegate, UITableVi
     let DEFAULT_POI_NAME: String = "POI"
     let DEFAULT_BUILDING_NAME: String = "Current Building"
     var lock = false
+    var stairs = GMSMarker()
```

Comment:
non serían os changeOfFloorMarkers? ou pintanse todalas escaleiras que hai no piso?

Reply:
Non podería haber rutas que nun mesmo piso teñan  máis dun cambio de planta